CMakeLists.txt

```diff
@@ -361,7 +361,10 @@ foreach (s ${COMPILETIME_OPTIONS_LIST})
 	set (COMPILETIME_OPTIONS "${COMPILETIME_OPTIONS} -D${s}")
 endforeach (s)
 
-generate_and_install_pkg_config_file(torrent-rasterbar libtorrent-rasterbar)
+# msys (on Windows) would still need pkg-config generated
```

Comment:
I don't think this answers the question: "why are we not generating a pkg-config file when building with msvc?"

Reply:
I've updated the comment.